app/views/my_facilities/registrations.html.erb

```diff
@@ -1,5 +1,3 @@
-<% if current_admin.feature_enabled?(:my_facilities_improvements) %>
-  <%= render "registrations_details_v2" %>
-<% else %>
+<% unless current_admin.feature_enabled?(:my_facilities_improvements) %>
```

Comment:
Do we need this guard here? Isn't it handled by the controller?

Reply:
That's a great point - thanks @kpethtel !